packages/nextjs-kit/src/types/index.ts

```diff
@@ -0,0 +1,14 @@
+export enum ASC {
+  ASC = 'ASC',
+  asc = 'asc',
+}
+export enum DESC {
+  DESC = 'DESC',
+  desc = 'desc',
+}
+
+export interface DataToSort {
+  data: Array<Record<string, string | number>>;
+  key: string;
+  direction: string;
+}
```

Comment:
I think this can be cleaned up a bit.

Note that we can use `Type[]` instead of `Array<Type>`, the latter of which is more difficult to read imo.
```suggestion
export interface DataToSort {
  data: Record<string, string | number>[];
  key: string;
  direction: 'asc' | 'ASC' | 'desc' | 'DESC';
}
```